tasks/main.yml

```diff
@@ -0,0 +1,24 @@
+---
+- name: Install "python-mysqldb" package
```

Comment:
what about the others?
`Install required packages` ?

Reply:
done